core/README.md

```diff
@@ -34,6 +34,22 @@ Run the API Tests locally
 pytest
 ```
 
+### Build and run in docker
+
+From the root of the repository, build the container image:
+
+```cmd
+cd core/api
+docker build -t tre-management-api .
+```
+
+To run:
+```cmd
+docker run -it -e PORT=8000 -p 8000:8000 tre-management-api
```

Comment:
What's the use case for having a variable internal port? Can we not fix the API to listen on 8000 and just allow them to provide a different mapping at the docker layer?

Reply:
Ah, this is left over from when linux web apps was in preview, they passed a PORT variable that varied (and still do - but is always 80 now unless otherwise specifed).

https://docs.microsoft.com/en-us/azure/app-service/configure-custom-container?pivots=container-linux#configure-port-number

I'll leave it to 8000, and set WEBSITES_PORT in the app service.